data/json/items/comestibles.json

```diff
@@ -1684,9 +1684,115 @@
     "weight": 225,
     "color": "white",
     "symbol": "%",
-    "description": "A bone from some creature or other.  Could be used to make some stuff, like needles.",
+    "description": "A bone from some creature or other.  Could be used to make some stuff, like needles, or cracked open for marrow.",
     "price": 0,
     "material": "bone",
+    "volume": 1,
+    "use_action": {
+      "target": "bone_marrow",
+      "msg": "You snap the bone into two pieces and extract the marrow.",
+      "menu_text": "Extract marrow",
```

Comment:
Missing `"target_charges": 2,` otherwise this will create [bone marrow item with 0 charges](https://cdn.discordapp.com/attachments/365907426270117888/474686597086838806/0bonemarrowcharges.png) and [prompt error when attempting to eat it](https://cdn.discordapp.com/attachments/365907426270117888/474686599494500374/bonemarrowerror.png).